src/FreeRTOSConfig.h

```diff
@@ -62,7 +62,7 @@
 #define configTICK_RATE_HZ                      1024
 #define configMAX_PRIORITIES                    (3)
 #define configMINIMAL_STACK_SIZE                (120)
-#define configTOTAL_HEAP_SIZE                   (1024 * 40)
```

Comment:
I'm guessing this needs the heap shrunk to build. Does the heap need to be shrunk by 5K though? That's a lot, if this uses that much memory then the design should probably be optimised. Would 39 work?

Reply:
Let's try it out. I do not understand, why the shrink is needed in the first place though. Do you?

Reply:
It's needed because this PR makes the stack larger, and so with the heap being this big, it can't fit. By decreasing the heap size, you allow more memory for the stack.

Reply:
Which means that there is currently no screen that needs a stack as large as this calculator?

Reply:
39 is sufficient. Thanks for the review!